packages/web-components/src/progressbar/progressbar.styles.ts

```diff
@@ -0,0 +1,156 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorBrandBackground2,
+  colorCompoundBrandBackground,
+  colorNeutralBackground6,
+  colorPaletteDarkOrangeBackground3,
+  colorPaletteGreenBackground3,
+  colorPaletteRedBackground3,
+} from '../theme/design-tokens.js';
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --progress-bar-thickness: 2px;
+    --progress-bar-radius: ${borderRadiusMedium};
+    --progress-bar-color: ${colorCompoundBrandBackground};
+    --progress-speed: 3s;
+
+    display: flex;
```

Comment:
You can remove `display` from here since you've defined it above using the `${display('flex')}` helper